manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/services/ProcessorServiceImpl.java

```diff
@@ -277,54 +283,64 @@ public Processor updateProcessorStatus(ResourceStatusDTO statusDTO) {
                     statusDTO.getGeneration());
             return processor;
         }
+
         Operation operation = processor.getOperation();
+        if (Objects.isNull(operation)) {
```

Comment:
@manstis Same comment here. I think this scenario is prevented by our DB schema.

Reply:
Removed check.